tasks/notify.py

```diff
@@ -678,6 +679,13 @@ def schedule_new_user_activated_task(self, org_ownerid, user_ownerid):
             args=None,
             kwargs=dict(org_ownerid=org_ownerid, user_ownerid=user_ownerid),
         )
+        # Activate the account user if it exists.
+        self.app.tasks[activate_account_user_task_name].apply_async(
+            kwargs=dict(
+                user_ownerid=user_ownerid,
+                org_ownerid=org_ownerid,
+            ),
+        )
```

Comment:
what's the difference between `celery_app.send_task()` and `self.app.tasks[activate_account_user_task_name].apply_async()`?

Reply:
I think the main difference between `send_task` and `apply_async` is the way to reference the task. `apply_async` requires the task to be registered on the existing process, whereas `send_task` requires only then name. Otherwise, they do the same thing (starts running a new task). I'm not entirely sure why `send_task` was used here. I didn't want to change the existing call, so I left it with `send_task`, but I _think_ that it can also be called with `apply_async`.